app/views/bulk_actions/_form.html.erb

```diff
@@ -60,6 +65,12 @@
           Reindexes the DOR object to Solr
         </span>
       </div>
+      <div role='tabpanel' class= 'tab-pane' id='manage_catkey_options'>
+        <span class='help-block'>
+          Adds or updates catkeys associated with objects.  You need two lists: druids and catkeys, and they need to correspond 1:1 in the correct order.
+        </span>
+        <%= render 'bulk_actions/forms/manage_catkey_form', f: f %>
```

Comment:
Why are you passing `f` and then not using it in the partial?

Reply:
oops